### PR TITLE
Added tdnf package manager

### PR DESCRIPTION
**Pull request by dthaluru**

### CODE REVIEW

**Comment by sofar** [comment]
:+1:

File: src/ccmodules/package_upgrade.c [hunk 1]
```diff
@@ -65,6 +65,8 @@ void package_upgrade_handler(GNode *node) {
 		exec_task("/usr/bin/dnf update --refresh");
 #elif defined(PACKAGE_MANAGER_APT)
 		exec_task("/usr/bin/apt-get upgrade");
+#elif defined(PACKAGE_MANAGER_TDNF)
+		exec_task("/usr/bin/tdnf update --refresh");
```
Comment (devimc) [inline src/ccmodules/package_upgrade.c]: does this command need the option --assumeyes ?

Comment (dthaluru) [inline src/ccmodules/package_upgrade.c]: Yes I need to add assumeyes parameter here too.

Comment (dthaluru) [inline src/ccmodules/package_upgrade.c]: Added assumeyes and updated the PR.

**Pull request merged**